content/post/GSOC_AMA_report.md

```diff
@@ -0,0 +1,109 @@
++++
+title = "ALiAS does it again: Report on our GSoC AMA program"
+slug = "aliasama"
+image = "images/logo.png"
+date = "2018-02-20T00:00:00"
+description = "A short report on ALiAS's successful Google Summer of Code program, and a brief story of how the idea came to life"
+disableComments = false
+author = "Sahil, Kaushlendra, Ayush, Vipul"
+categories = ["New Horizons"]
+tags = ['GSoC', 'Community', 'Google', 'AMA', 'FOSS', 'Amity', 'University']
```

Comment:
Amity and University are supposed to be different tags?

Reply:
Yes, blogging is weird, and something that I have found to be quite effective is to break down each word. This leads to more hits. 
Example.
I search for: 
1. GSoC alias amity - 2 hits 
2. Amity university community - 3 hits
More hits with tags, more probable for Google to show you on the top.

Reply:
I'll use that for my blog then xD